testing/test_main.py

```diff
@@ -0,0 +1,142 @@
+import os
+import sys
+from fastapi.testclient import TestClient
+import pytest
+import random
+import string
+
+
+from .main import app, frankenbert
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+client = TestClient(app)
+
+""" Testing all Gets"""
+
+def test_get_bot_scripts():
+    response = client.get("/select-all-from-bot-scripts")
+    assert response.status_code == 200
+
+def test_frankenbert():
+    test = 'A women walked her dog down the street and cop waved to her'
+    response = client.get(f'/frankenbert/{test}')
+    assert response.status_code == 200
+    response_body = response.json()
+    assert 'Rank' in response_body['Rank'] 
+    assert response_body['Rank'][5] in ['0', '1', '2', '3', '4', '5']
+    assert '%' in response_body['Confidence'] 
+
+
+def test_view_data():
+    response = client.get('/view-data')
+    assert response.status_code == 200
+    response_body = response.json()
+    assert len(response_body) == 5000
+
+
+def test_to_approve():
+    response = client.get('/view-data')
+    assert response.status_code == 200
+
+
+def test_advance_all():
+    response = client.get('/advance-all')
+    assert response.status_code == 200
+
+
+"""Test all Posts"""
+
+
+def test_form_in():
+    response = client.post(
+        '/form-in/',
+        json={
+                "city": "cHicago",
+                "state": "ILLInoIs",
```

Comment:
Not sure what happened with location here, but I don't think case sensitivity will play an issue.

Reply:
It was a test to make sure that case sensitivity would not be an issue in the form as the API docs do not specify case sensitivity.  However future iterations should be done with a random string generator.

Reply:
Its nice. I don't think the code base currently has any test. We need more. I approve of tests.

Reply:
These tests all look great. Case sensitivity test is something small but could cause big problems if not accounted for.